documentation/resource_spotlight.md

```diff
@@ -0,0 +1,87 @@
+spotlight
+===
+
+Use the **spotlight** resource to manage the metadata indexing state for disk volumes. This
+will primarily affect the ability to search volume contents with the macOS Spotlight feature.
+Under the hood, a **spotlight** resource executes the `mdutil` command in the `metadata_util`
+library.
+
+[Learn more about Spotlight](https://support.apple.com/en-us/HT204014).
+
+[Learn more about the `mdutil` command](https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/mdutil.1.html).
+
+Syntax
+------
+
+The most basic usage of a **spotlight** resource block declares a disk volume as the name property
+to **enable** metadata indexing:
+
+```ruby
+spotlight '/'
+```
+
+The full syntax for all of the properties available to the **spotlight** resource is:
+
+```ruby
+spotlight 'volume name' do
+  volume                      String # defaults to 'volume name' if not specified
+  indexed                     TrueClass, FalseClass # defaults to TrueClass if not specified
+  allow_search                TrueClass, FalseClass # defaults to TrueClass if not specified
```

Comment:
Similar here: I think it might be better to have this be `searching_disabled` and match the same sort of format for the boolean property.

Reply:
I've updated this to `searchable` for clarity.